proxy/api/src/config.rs

```diff
@@ -21,10 +19,11 @@ pub enum Error {
     MissingExePath(path::PathBuf),
 }
 
-/// Returns the directories to locate all application state.
-#[must_use]
-pub fn dirs() -> ProjectDirs {
-    ProjectDirs::from("xyz", "radicle", "radicle-upstream").expect("couldn't build dirs")
+/// Returns the directories to locate all application state specific to the current identity.
+pub fn id_dirs() -> Result<path::PathBuf, Error> {
+    let home_dir = std::env::var("HOME")?;
+
+    Ok(path::Path::new(&home_dir).join(".radicle/identities/current"))
```

Comment:
Actually, now that I'm thinking about it, why did we opt for `~/.radicle` instead of putting `identities` under the project dir?

Reply:
My thinking was that since the git helper has to be in `~/.radicle/bin` anyway (because of https://github.com/radicle-dev/radicle-upstream/issues/763#issuecomment-671374819), we could have everything in one place. Would also be easier to instruct users, because the directory would be in the same place for every platform.

I know this is not _the right way_, so I'm open for counter-arguments.

Reply:
This would make it also easier to back up or remove all radicle data, since it wouldn't be spread around the filesystem: https://github.com/radicle-dev/radicle-upstream/blob/master/scripts/reset-state.sh.

Reply:
Alrighty, thanks for the clarification :+1: